### PR TITLE
add release notes for 1.14.0

### PR DESCRIPTION
**Pull request by j-windsor**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,39 @@
...
 This page documents production updates to Cloud Code for Visual Studio Code. You can check this page for announcements about new or updated features, bug fixes, known issues, and deprecated functionality.
 
+## Version 1.14.0 (Sept 2021)
+
+### New Features
+
+* **New and Improved Kubernetes Development Experience:** Your Kubernetes development sessions now have a vastly improved experience with the new Development Sessions explorer!
+![Development Sessions Welcome View](https://www.gstatic.com/cloudssh/cloudcode/welcome-development-sessions.png)
+    * See exactly which task is executing and the status of each task.
+    ![Development Sessions In Progress](https://www.gstatic.com/cloudssh/cloudcode/in-progress-development-sessions-1.png) ![Development Sessions In Progress](https://www.gstatic.com/cloudssh/cloudcode/in-progress-development-sessions-2.png)
```
Comment (kschaab) [inline CHANGELOG.md:12]:
```suggestion
    ![Development Sessions In Progress](https://www.gstatic.com/cloudssh/cloudcode/in-progress-development-sessions-2.png)
```

**Review by kschaab — APPROVED** [review]
LGTM with suggestion

**Review by Caglar1881 — COMMENTED** [review]
#450

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,39 @@
 
 This page documents production updates to Cloud Code for Visual Studio Code. You can check this page for announcements about new or updated features, bug fixes, known issues, and deprecated functionality.
 
+## Version 1.14.0 (Sept 2021)
```
Comment (Caglar1881) [inline CHANGELOG.md:5]: ## Sürüm 1.14.0

**Review by Caglar1881 — COMMENTED** [review]
#450

**Pull request merged**